src/xrpld/app/tx/detail/Escrow.cpp

```diff
@@ -579,17 +618,12 @@ EscrowFinish::doApply()
         std::vector<uint8_t> wasm(wasmStr.begin(), wasmStr.end());
         std::string funcName("ready");
 
-        auto const escrowTx =
-            ctx_.tx.getJson(JsonOptions::none).toStyledString();
-        auto const escrowObj =
-            slep->getJson(JsonOptions::none).toStyledString();
-        std::vector<uint8_t> escrowTxData(escrowTx.begin(), escrowTx.end());
-        std::vector<uint8_t> escrowObjData(escrowObj.begin(), escrowObj.end());
-
         WasmHostFunctionsImpl ledgerDataProvider(ctx_, k);
 
-        std::uint32_t gasLimit = ctx_.app.config().FEES.extension_compute_limit;
-        auto re = runEscrowWasm(wasm, funcName, &ledgerDataProvider, gasLimit);
+        std::uint32_t allowance = std::min(
```

Comment:
This already checked in preflight, you can just use `sfComputationAllowance`

Reply:
Good catch